docs/assign_sla.md

```diff
@@ -6,22 +6,27 @@ def assign_sla(object_name, sla_name, object_type, timeout=30)
+| object_type                     | str  | The Rubrik object type you want to assign to the SLA Domain.                                                                                                                                                                                               | vmware, mssql_host |
+| log_backup_frequency_in_seconds | int  | (Optional) The MSSQL Log Backup frequency you'd like to specify with the SLA. Required when the `object_type` is `mssql_host`.                                                                                                                             |                    |
+| log_retention_hours             | int  | (Optional) The MSSQL Log Retention frequency you'd like to specify with the SLA. Required when the `object_type` is `mssql_host`.                                                                                                                          |                    |
+| copy_only                       | bool | (Optional) Take Copy Only Backups with MSSQL. Required when the `object_type` is `mssql_host`.                                                                                                                                                             |                    |
 ## Keyword Arguments
-| Name        | Type | Description                                                                 | Choices | Default |
-|-------------|------|-----------------------------------------------------------------------------|---------|---------|
-| timeout  | str  | The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error.  |         |    30     |
+| Name    | Type | Description                                                                                                  | Choices | Default |
+|---------|------|--------------------------------------------------------------------------------------------------------------|---------|---------|
+| timeout | str  | The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. |         | 30      |
 
 ## Returns
-| Type | Return Value                                                                                   |
-|------|-----------------------------------------------------------------------------------------------|
+| Type | Return Value                                                                                           |
+|------|--------------------------------------------------------------------------------------------------------|
 | str  | No change required. The vSphere VM '`object_name`' is already assigned to the '`sla_name`' SLA Domain. |
```

Comment:
Probably worth changing the text `vSphere VM` to be either a variable with both `vSphere VM` or `MSSQL Host`.

Reply:
Since the `No change required. The MSSQL Instance` message is unique enough I added it as a possible return value in the docs.